### ISSUE TITLE
[Bug]: Requirement that entry_point is below does not work when entry_point is in node_modules package

### ISSUE THREAD

**Issue by twheys** [aspect-build/rules_webpack#117]
### What happened?

My build has a production optimization step that uses webpack to bundle some node_modules packages. We use the webpack_bundle rule to do this, but in the latest version, 0.12.0, a validation error is raised preventing this, which requires that the entrypoint is relative to the build package.

```
Error in fail: Path 'node_modules/.aspect_rules_js/styletron-client@3.0.4/node_modules/styletron-client' is not beneath 'horizon'
```

I understand the motivation for the validation step but perhaps an exception can be made for `node_modules/.aspect_rules_js`

### Version

rules_webpack 0.12.0

**Comment by jbedard** [comment]
Is this issue still relevant?
